CHANGELOG.rst

```diff
@@ -7,7 +7,8 @@ v1.4.0 (??.??.2024)
 
 Feature
 -------
-- Changed supported experiment configuration file type to YAML. 
+- Changed supported experiment configuration file type to YAML.
+- Add support for ssh jump hosts in the database connection.
```

Comment:
Additionally: 
- Changed supported database configuration file type to YAML.

Reply:
I adapted this, but different then mentioned by you